src/Microsoft.DotNet.Darc/src/Darc/Operations/AddSubscriptionOperation.cs

```diff
@@ -177,6 +195,19 @@ public override async Task<int> ExecuteAsync()
                                                                            batchable,
                                                                            mergePolicies);
                 Console.WriteLine($"Successfully created new subscription with id '{newSubscription.Id}'.");
+
+                // Prompt the user to trigger the subscription unless they have explicitly disallowed it
+                if (!_options.NoTriggerOnCreate)
+                {
+                    bool triggerAutomatically = _options.TriggerOnCreate;
+                    triggerAutomatically = triggerAutomatically || UxHelpers.PromptForYesNo("Trigger this subscription immediately?");
```

Comment:
[nit] can probably move the or to the previous line. I don't think splitting this makes adds much to readability.